test/unit/mbyte_spec.lua

```diff
@@ -10,6 +10,7 @@ local charset = helpers.cimport('./src/nvim/charset.h')
 describe('mbyte', function()
   if helpers.isCI('quickbuild') then
     pending("crashes on quickbuild", function() end)
+    return
```

Comment:
Oh, confusing.  For reference: https://github.com/Olivine-Labs/busted/issues/610